server/lib/routes/get-verify-email.js

```diff
@@ -0,0 +1,47 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const got = require('got');
+
+const logger = require('mozlog')('server.get-verify-email');
+const config = require('../configuration');
+
+module.exports = function (uid, code, options) {
```

Comment:
`uid, code, options` arguments here won't be useful. All these come from `req` below